Code/Tools/ProjectManager/Platform/Mac/ProjectManagerDefs_mac.cpp

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ProjectManagerDefs.h>
+
+
+namespace O3DE::ProjectManager
+{
+    static const QString ProjectBuildPathPostfix = ProjectBuildDirectoryName + "/mac";
```

Comment:
I think we should use `mac_xcode` for the build folder to indicate the generator.

Reply:
same here, this value is consistent with ci_build